packages/frontend/graphql/GetStudentsQuery.gql

```diff
@@ -0,0 +1,12 @@
+
+query getStudents {
+	students {
+    nodes { 
+    	id
+      name
+      key
+      createdAt
+    }
+  }
+	
+}
```

Comment:
Rename this file to `operations.gql` in this single file we will write all our graphql custom queries